cdk/PgStacInfra.ts

```diff
@@ -5,6 +5,8 @@ import {
   aws_iam as iam,
   aws_ec2 as ec2,
   aws_rds as rds,
+  aws_lambda as lambda,
+  aws_iam,
```

Comment:
already imported as `iam`

Reply:
thanks, will be addressing these minor fixes in a following PR